apps/web-giddh/src/app/vouchers/create/create.component.ts

```diff
@@ -716,15 +741,68 @@ export class VoucherCreateComponent implements OnInit, OnDestroy, AfterViewInit
         /** Voucher details */
         this.componentStore.voucherDetails$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
             if (response) {
-                this.getAccountDetails(response?.account.uniqueName);
+                this.getAccountDetails(response?.account?.uniqueName);
```

Comment:
If we already check response in first, then we will remove optional chaining check in response in overall subscribe.